dev/build/windows/patches_coq/lablgtk-2.18.3.patch

```diff
@@ -10,10 +16,8 @@ diff -u -r lablgtk-2.18.3/configure lablgtk-2.18.3.patched/configure
    { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: Ignoring ocamlfind" >&5
  $as_echo "$as_me: WARNING: Ignoring ocamlfind" >&2;}
    OCAMLFIND=no
-
-diff -u -r lablgtk-2.18.3/src/glib.mli lablgtk-2.18.3.patched/src/glib.mli
```

Comment:
Is removing the diff -u lines intentional?

Reply:
More or less yes. I use a script to compare tar files with folders now to create patches to tar files (difftar-folder.sh in /dev/build/windows folder). This script doesn't produce these lines cause of the way diff is called, but of cause I might be able to change the script. I created this script, because it was a bit unreliable/inconvenient to create patches to tar files in other ways.

The information is anyway redundant. The two change lines below give the complete information on the patch (including if it is unified (-u) format or not).

I added an additional header (which is ignored by patch) which details how the script was called.